tests/test_games_do_not_fail.py

```diff
@@ -44,6 +44,21 @@ def test_objects_game():
         ),
     )
 
+def test_basic_games_game():
+    run_game(
+        "egg.zoo.basic_games.play",
+        dict(
+            mode="gs",
+            train_data="egg/zoo/basic_games/data_generation_scripts/example_reconstruction_input.txt",
+            validation_data="egg/zoo/basic_games/data_generation_scripts/example_reconstruction_input.txt",
```

Comment:
any reason for not testing the discrimination game? I guess they share most of the code so maybe not really worth it, right?

Reply:
The games are very similar, and testing is a bit slow, so I would just test one.